backend/typescript/models/camp.model.ts

```diff
@@ -35,134 +35,139 @@ export interface Camp extends Document {
+  {
+    active: {
+      type: Boolean,
+      required: true,
+    },
+    ageLower: {
+      type: Number,
+      required: true,
+    },
+    ageUpper: {
+      type: Number,
+      required: true,
+    },
+    campCoordinators: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "User",
+        },
+      ],
+      default: [],
+    },
+    campCounsellors: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "User",
+        },
+      ],
+      default: [],
+    },
+    campSessions: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "CampSession",
+        },
+      ],
+      default: [],
+    },
+    description: {
+      type: String,
+    },
+    dropoffFee: {
+      type: Number,
+      required: true,
+    },
+    pickupFee: {
+      type: Number,
+      required: true,
+    },
+    endTime: {
+      type: String,
+      required: true,
+    },
+    earlyDropoff: {
+      type: String,
+    },
+    fee: {
+      type: Number,
+      required: true,
+    },
+    fileName: {
+      type: String,
+    },
+    formQuestions: {
+      type: [
+        {
+          type: Schema.Types.ObjectId,
+          ref: "FormQuestion",
+        },
+      ],
+      default: [],
+      required: true,
+    },
+    latePickup: {
+      type: String,
+    },
+    location: {
+      streetAddress1: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-  },
-  campCounsellors: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "User",
+      streetAddress2: {
+        type: String,
+        required: false,
       },
-    ],
-    default: [],
-  },
-  campSessions: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "CampSession",
+      city: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-  },
-  description: {
-    type: String,
-  },
-  dropoffFee: {
-    type: Number,
-    required: true,
-  },
-  pickupFee: {
-    type: Number,
-    required: true,
-  },
-  endTime: {
-    type: String,
-    required: true,
-  },
-  earlyDropoff: {
-    type: String,
-  },
-  fee: {
-    type: Number,
-    required: true,
-  },
-  fileName: {
-    type: String,
-  },
-  formQuestions: {
-    type: [
-      {
-        type: Schema.Types.ObjectId,
-        ref: "FormQuestion",
+      province: {
+        type: String,
+        required: true,
       },
-    ],
-    default: [],
-    required: true,
-  },
-  latePickup: {
-    type: String,
-  },
-  location: {
-    streetAddress1: {
+      postalCode: {
+        type: String,
+        required: true,
+      },
+    },
+    name: {
       type: String,
       required: true,
     },
-    streetAddress2: {
+    campProductId: {
       type: String,
-      required: false,
     },
-    city: {
+    dropoffPriceId: {
       type: String,
-      required: true,
     },
-    province: {
+    dropoffProductId: {
```

Comment:
probably not related to this ticket but I saw that we have all of the following fields: 
1. dropoffFee
2. pickupFee
3. dropoffPriceId
4. dropoffProductId
5. pickupPriceId
6. pickupProductId

Are all these fields required for Stripe payments?

Reply:
I think @annechung would know for this one! Although I do think this is for Stripe payments